recipes/tcb-span/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, tools
+import os
+
+
+class TcbSpanConan(ConanFile):
+    name = "tcb-span"
+    description = "Implementation of C++20's std::span for older C++ standards"
+    topics = ("conan", "span", "header-only", "tcb-span")
+    homepage = "https://github.com/tcbrindle/span"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSL-1.0"
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
You are right we can remove the extra settings here

```suggestion
    settings = "compiler"
```

Reply:
I guess we still need the `compiler` setting because of the c++ minimum version requirement, right?

Reply:
yes!